Sources/HaCWebsiteLib/Events/EventManager.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+
+public struct EventManager {
+    private(set) static var events: [Event] = []
+
+    public static func update() throws{
+        var updatedEvents : [Event] = []
+
+        try! WorkshopManager.update()
+        //TODO: source events from database
+        let workshopEvents : [Event] = WorkshopManager.workshops.map{
+            (workshop: Workshop) -> WorkshopEvent in
+                return WorkshopEvent(called: workshop.title, at: Date(), described: "There's Pizza", 
+                    coloured: "purple", hypePeriod: 7, coolOffPeriod: 7, basedOn: workshop)
+        }.filter{ event in 
+            event.isLive()
+        }
+        updatedEvents += workshopEvents
+
+        updatedEvents += [HackathonEvent(called: "Game Gig 2017", at: Date(), described: "Fun and Games", 
+            coloured: "black", hypePeriod: 7, coolOffPeriod: 7, lasting: 3)]
+
+        events = updatedEvents
+    }
+}
+
+extension Event {
+    func isLive() -> Bool {
```

Comment:
Nit: It's generally more Swifty to have computed properties over functions that don't have arguments (as long as the complexity is `O(1)`)

Reply:
Another note. I feel like `isLive` doesn't communicate what we're doing here. To me, it says "is this event going ahead or not, have we launched, is it published?". I imagine it is still possible to link to events even if this `isLive()` returns false.

Possibly `shouldShowAsUpdate` is a better name?